security/certora/specs/Governance.spec

```diff
@@ -725,7 +773,7 @@ rule proposal_executes_after_cooldown_period(){
 //              From its perspective, execution is sent to a Portal.
 //Property #18: VOTING_TOKENS_CAP in GovernanceCore should be big enough to account for tokens that need to pass multiple slots,
 //               and big enough for at least mig to long term
-
+// ND - what are the above 16 - 18 properties? 
```

Comment:
Property #17 We summarize `CrossChainForwarder.forwardMessage` to NONDET. This function sends payloads to execution chain, and it's called by `executeProposal()`. I added a comment about the summarization to the "assumptions and simplifications" section of the report. So the property seems to hold.
Property #18: VOTING_TOKENS_CAP is not relevant anymore, the constant is not used now. I'll notify the customer.